### PR TITLE
Mobile form section

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

- Added Email field
- Added Message field that accepts up to 500 chars.
- Added fields have placeholder text.
- Added name field that accepts up to 30 chars.
- Collect data by using the Formspree service
- Implemented the button interactions

### CODE REVIEW

**Review by V-Blaze — CHANGES_REQUESTED** [review]
Hi @Bleu-Yves-Sopoude  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project, that does not have a linter check, Your linters are not running properly. I'll suggest you go through your project and set up the linter properly, You can always get the linters setup process from Microverse [here](https://github.com/microverseinc/linters-config). :smile: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -163,6 +213,70 @@ <h2>About</h2>
...
+      <div class="form-background"></div>
+
+      <div class="get-started"> 
+
+      <h2>Get started</h2>
+  <p>
+    I'm always interested in hearing about <br />
+    new projects, so if you'd like to chat <br />
+    please get in touch.
+  </p>
```
Comment (mahbubul14) [inline index.html]: - [ ] To improve the alignment of the get started section, it's recommended to adjust the layout of the 'Get Started' section by adding a left margin and ensuring that the font, colors, and spacing match the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign | Figma

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Bleu-Yves-Sopoude,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 2]
```diff
@@ -163,6 +213,70 @@ <h2>About</h2>
...
+                  <label>
+                      Your message:
+                  </label>
+                  <textarea  name="message" rows="5" placeholder="Enter your message here..." maxlength="500" required></textarea>
+              </div>
+             <div class="input-group">
+              <button type="submit" class="input-form">Start collaboration</button>
+             </div>
+          </form>
+     </section>
```
Comment (GedeonTS) [inline index.html]: - Good job implementing all the funtionalities, however, just as requested by your last reviewer [here](https://github.com/Bleu-Yves-Sopoude/mb-portfolio/pull/4#discussion_r1192077778), kindly make sure your Design matches the Figma design as much as possible according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma). PLease pay close attention to margins, spaces, sizes of your placeholders and fonts (family and sizes)

**Figma Design**
Screenshot from 2023-05-12 15-18-21

**Your Design**
Screenshot from 2023-05-12 15-19-31

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Bleu-Yves-Sopoude, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- Good PR Description :+1: 
- No Linters Errors :100: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @Bleu-Yves-Sopoude   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Descriptive pr👍 
- Linters are passing 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**